app/pages/reporter/ciip-application.tsx

```diff
@@ -41,8 +50,23 @@ class CiipApplication extends Component<Props> {
   });
 
   render() {
-    const {query} = this.props;
+    const {query, router} = this.props;
     const {session} = query || {};
+    const {application} = query || {};
+
+    if (
+      application &&
+      application.latestDraftRevision.legalDisclaimerAccepted === false
```

Comment:
lets use optional chaining here. I'm actually not sure why this wasn't picked up by the linter
```suggestion
      application?.latestDraftRevision?.legalDisclaimerAccepted === false
```

Reply:
Upated